client/app/components/groupInfo/groupInfo.html

```diff
@@ -2,7 +2,7 @@
   <md-toolbar class="md-hue-1">
     <div class="md-toolbar-tools">
       <h1 class="md-title" flex>
-        <md-button ui-sref="groupList" class="md-icon-button">
+        <md-button ui-sref="joinGroupList" class="md-icon-button">
```

Comment:
Yes I was also thinking about that. Let me now how I can also contribute to the foodsaving-e2e repository.